src/components/Book.js

```diff
@@ -1,32 +1,62 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
 import styles from '../styles/Book.module.css';
 import Button from './Button';
 import { removeBook, deleteBook } from '../redux/books/booksSlice';
+import 'react-circular-progressbar/dist/styles.css';
 
-const Book = ({ title, author, id }) => {
+const Book = ({
+  title, author, category, id,
+}) => {
   const dispatch = useDispatch();
+
+  const progress = Math.floor(Math.random() * 100);
   return (
     <div className={styles.book}>
-      <h2>{title}</h2>
-      <p>{author}</p>
-      <Button
-        type="button"
-        title="Remove"
-        onClick={
-        () => {
-          dispatch(removeBook({ id }));
-          dispatch(deleteBook(id));
-        }
-    }
-      />
+      <div className={styles.info}>
+        <p className={styles.category}>{category}</p>
+        <h2 className={styles.title}>{title}</h2>
+        <p className={styles.author}>{author}</p>
+        <div className={styles.buttonsDiv}>
+          <Button
+            className="book-action"
+            type="button"
+            title="Remove"
+            onClick={
+              () => {
+                dispatch(removeBook({ id }));
+                dispatch(deleteBook(id));
+              }
+            }
+          />
+          <span className={styles.separator}>|</span>
+          <Button
+            className="book-action"
+            type="button"
+            title="Comment"
+          />
+          <span className={styles.separator}>|</span>
+          <Button
+            className="book-action"
+            type="button"
+            title="Edit"
+          />
+        </div>
+      </div>
+      <div className={styles.progress}>
+        <div className={styles.progressCircle}>
+          <CircularProgressbar value={progress} text={`${progress}%`} />
+        </div>
+      </div>
     </div>
   );
 };
```

Comment:
After comparing your project with the given template from Zeplin, I notice that some discrepancies as shown below 👇🏾 
_**Please note that the missing part is marked in red on the Zeplin design**_

Kindly implement this.

Zeplin Design | Your project
---- | ----
![image](https://user-images.githubusercontent.com/84156714/222126260-3a447896-06f1-4738-a26d-8418a6bd711e.png) | ![image](https://user-images.githubusercontent.com/84156714/222126375-7d40cc64-11ff-4447-8495-179569b0740e.png)